### DISCUSSION TITLE
Вітання в обговореннях publisher-slack-bot!

### DISCUSSION THREAD

**Discussion by Andygol** [k3ilona/publisher-slack-bot#1] [Announcements]
## 👋 Вітання!
  Ми використовуємо обговорення як місце для спілкування з іншими членами нашої спільноти. Ми сподіваємося, що ви будете:
  * Ставити питання, які вам цікаві.
  * Ділитись ідеями.
  * Взаємодіяти з іншими членами спільноти.
  * Вітати інших та відкриті до спілкування. Це спільнота, яку ми 
  будуємо разом 💪.

  Для початку, ви можете залишити коментар з відомостями про себе, щоб розповісти іншим про те як ви берете участь в житті спільноти.

**Comment by vit-um** [comment]
Вітаю всіх! Я взявся за Слак бота, зробим вже API-ключ, завів Workspace у Slack та запросив до нього усіх членів команди. Саме в цьому  Workspace буде діяти наш бот в межах каналу #hackathon
За пару днів спробую довести код до якогось біль-менш робочого стану та запакувати його в контейнер.

**Comment by vit-um** [comment]
Треба основну нашу репу з застосунком періменовувати на `Ilonabot`  
За правилами розробки на мові го, назва репозиторію має відповідати назві застосунку.  У нас так:
`github.com/k3ilona/publisher-slack-bot`

```sh
✗ ./publisher-slack-bot version
v1.0.0
```
